lib/amazon_ssa_support/amazon_ssa_extract.rb

```diff
@@ -0,0 +1,174 @@
+#$:.push("#{File.dirname(__FILE__)}/..")
```

Comment:
@Fryguy What are the conventions for top-level scripts in a GEM? Given the standard conventions, one would expect a class or module named `AmazonSsaExtract` to be defined in this file. Should files like this be under `lib`, or should they be in a `bin` or `scripts` directory of the GEM.